DEFS

```diff
@@ -76,6 +76,9 @@ def java_test(
       # http://arihantwin.blogspot.com/2012/08/getting-error-illegal-local-variable.html
       '-XX:-UseSplitVerifier',
 
+      # Don't use the system-installed JNA; extract it from the local jar.
+      '-Djna.nosys=true',
+
```

Comment:
That's a little bit terrifying.  I wonder if this would also help fix https://github.com/facebook/buck/issues/638